js/app/filters/wordCount.js

```diff
@@ -0,0 +1,10 @@
+app.filter('wordCount', function () {
+	'use strict';
+	return function (value) {
+		if (value && (typeof value === 'string')) {
+			return value.trim().split(/\s+/).length;
```

Comment:
Simple idea but also counts characters such as * an # which are pretty frequent in markdown